keyboards/tzarc/kobold/kobold.c

```diff
@@ -0,0 +1,18 @@
+// Copyright 2025 Nick Brassel (@tzarc)
+// SPDX-License-Identifier: GPL-2.0-or-later
+#include QMK_KEYBOARD_H
+
+void board_init(void) {
+    // Disable RGB LEDs
+    gpio_set_pin_output(RGB_ENABLE_PIN);
+    gpio_write_pin(RGB_ENABLE_PIN, 1);
+}
```

Comment:
Does this need to be done super early in `board_init`? While not consistent, I think we were aiming for `board_init` to be used more for the port remap style stuff.

```suggestion
void keyboard_pre_init_kb(void) {
    // Disable RGB LEDs
    gpio_set_pin_output(RGB_ENABLE_PIN);
    gpio_write_pin(RGB_ENABLE_PIN, 1);
    
    void keyboard_pre_init_user();
}
```

Reply:
It's mainly due to a design flaw where the RGB enable didn't include a pullup resistor. Moved it to `early_hardware_init_post` (which is earlier, now that I think about it) -- if you still want it in `keyboard_pre_init_kb` let me know.